src/components/ButtonMultiline.tsx

```diff
@@ -59,13 +59,18 @@ export const ButtonMultiline = ({
       }
     : {};
   const externalArrowIcon = textColor === palette.white ? 'icon-external-arrow-light' : 'icon-external-arrow';
-
+  const borderRadius = 4;
```

Comment:
Those single and multi-line buttons are treated like `InfoButton` with a border radius of 10